index.html

```diff
@@ -24,16 +24,15 @@
   <body>
     <div class="container">
       <div class="header">
-        <ul class="nav nav-pills pull-right">
-          <!--<iframe src="http://ghbtns.com/github-btn.html?user=skycocker&repo=chromebrew&type=watch&size=large"
-            allowtransparency="true" frameborder="0" scrolling="0" width="80" height="30"></iframe>-->
 
-          <a href="https://github.com/skycocker/chromebrew"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
+          <a href="https://github.com/skycocker/chromebrew"><img style="position: absolute; top: 0; right: 0; border: 0;" src="forkme.png" alt="Fork me on GitHub"></a>
         </ul>
-        <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="jumbotron">
+        <div class="center">
+          <img src="brew.png" alt="centered image" width="139.7142857143" hight="167.4285714286">
```

Comment:
Please remove `width` and `height` properties and resize the image to the correct size instead.  You have `hight` misspelled also.

Reply:
The image is still a bit jagged.  @superloach, can you please help him with this?